server/src/controllers/exercises.controller.js

```diff
@@ -1,5 +1,5 @@
-(req, res) {
+function getExercise(req, res) {
   Exercise.find()
-  .then((exercise) => res.json(exercise))
-  .catch((err) => console.log('Route error 404'))
-}
\ No newline at end of file
+    .then((exercise) => res.json(exercise))
+    .catch((err) => console.log('Route error 404'));
```

Comment:
```suggestion
    .catch((err) => res.status(404).send("some message");
```

Always send a response or the request will timeout, resulting in a network error.